server/start.go

```diff
@@ -147,6 +148,8 @@ which accepts a path for the resulting pprof file.
 	cmd.Flags().Uint64(FlagHaltTime, 0, "Minimum block time (in Unix seconds) at which to gracefully halt the chain and shutdown the node")
 	cmd.Flags().Bool(FlagInterBlockCache, true, "Enable inter-block caching")
 	cmd.Flags().String(flagCPUProfile, "", "Enable CPU profiling and write to the provided file")
+	cmd.Flags().String(flagSDKPprofAddr, "", "Enable SDK specific pprof server that starts independtly from Tendermint."+
```

Comment:
Generally not a fan of long flag descriptions. Consider putting this in the `Long` section of the command.